app.rb

```diff
@@ -0,0 +1,137 @@
+require_relative 'book'
+require_relative 'classroom'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
```

Comment:
![image](https://github.com/noelfoka/library-school/assets/93039359/a884335c-f2cb-4277-bf58-da8e257edbf1)
![image](https://github.com/noelfoka/library-school/assets/93039359/544ebf7f-0f53-4486-840d-39ff506030b2)

- [ ] Again it would be great if you replaced the colon with the equal sign at both places in the constructor to fix the error of your app 👍

Reply:
This change has been done well